_posts/kubernetes/2018-05-17-kubernetes-v2.md

```diff
@@ -0,0 +1,13 @@
+---
+date: 2018-6-27
+title: Using the Kubernetes V2 Provider
+categories:
+   - Kubernetes
+description: Kubernetes V2 Provider
+type: Document
+---
+06/27/2018
+
+This video explains how to use the Kubernetes V2 Provider with manifestes stored in Github.
+
+<video src="https://dha4w82d62smt.cloudfront.net/items/302F3G0Q082V3z123V43/Screen%20Recording%202018-06-27%20at%2010.06%20AM.mov" controls style="display: block;height: auto;width: 100%;">Screen Recording 2018-06-27 at 10.06 AM.mov</video>
```

Comment:
would you be able to upload it to youtube so it's like our other videos? 
like: https://github.com/armory/knowledge-base/blame/master/_posts/getting-started-video-series/1000-01-01-pipelines-part-1.md